src/AddTodoForm.jsx

```diff
@@ -1,8 +1,15 @@
-function AddTodoForm () {
+function AddTodoForm (props) {
+    function handleAddTodo(event) {
+       event.preventDefault();
+       let todoTitle = event.target[0].value;
```

Comment:
I'm not really sure why this works, I think it's some JavaScript bullshit.

If I do `console.log(typeof event.target)` I get `object`. For some reason when you treat it like an array here and get the 0th element, that returns you the first element inside the form (which is the object you get from `event.target`, since that's what's triggering the event).

I think what the instructions WANT you to do here is use `event.target.title`, which will work once we set `name="title"` on the input (see the other comment below). Given that this works though and we're going to remove it in the next assignment, it's not necessary to fix it here. Just wanted to clarify the intention.